en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md

```diff
@@ -1,118 +1,163 @@
 # Monitoring Server Health
 
-!!! warning
-This document is work in progress!
+**Health Check API** is used to validate and verify the status of the APIM server and its dependencies
+It is a REST service which returns the operational status, performance information of the configured endpoints with appropriate HTTP return code and a JSON object
+There are two health checkers available by default:
 
-
-The Carbon Health Check API can be used to check the health of a Carbon server. The sections below guide you through using this API.
-
--   [Deploying the API](#MonitoringServerHealth-DeployingtheAPI)
--   [Configuring the API](#MonitoringServerHealth-ConfiguringtheAPI)
--   [Invoking the API](#MonitoringServerHealth-InvokingtheAPI)
--   [Error responses](#MonitoringServerHealth-Errorresponses)
--   [Adding new health checkers](#MonitoringServerHealth-Addingnewhealthcheckers)
+1.  **Data sources health checker** - This checker goes through the data sources that are configured in the `<API-M_HOME>/repository/conf/deployment.toml` file and checks if the active connection count surpasses a healthy percentage limit (e.g., 80%) of the maximum allowed connections count. This checker also tests the connection from each data source to see whether the connection is successful.
+2.  **User store health checker** - This checker iterates through configured user stores of the super tenant domain and attempts to invoke the `isExistingUser` method to check whether a failure occurs.
```

Comment:
checks for which user? failure indicating the user is not in the user store?

Reply:
The tool will create a dummy user role in the configured user store manager and try to do the validation. But as it corresponds to the implementation , we can remove this.